db/migrate/20160610151323_create_posts.rb

```diff
@@ -0,0 +1,9 @@
+class CreatePosts < ActiveRecord::Migration
+  def change
+    create_table :posts do |t|
+      t.string :title
```

Comment:
Your `Post` model specifies that both `title` and `url` must be present. Would it be worth ensuring this also at the database level? With Rails, it's possible to modify records without running validations in certain use-cases, in which case your database would have `NULL` here which would cause an exception when trying to read this value in the view.

You can make it so that the database MUST have data on a particular column by adding `t.string :title, null: false`